### ISSUE TITLE
[BUG/ISSUE] Typos in state_diag_mod.F90?

### ISSUE THREAD

**Issue by FeiYao-Edinburgh** [geoschem/geos-chem#474]
Hi there,

I found the following snippet in Headers/state_diag_mod.F90 (GC v12.9.3):
```
    State_Diag%Archive_AOD  = ( State_Diag%Archive_AODHygWL1            .or. &
                                State_Diag%Archive_AODHygWL2            .or. &
                                State_Diag%Archive_AODHygWL3            .or. &
                                State_Diag%Archive_AODSOAfromAqIsopWL1  .or. &
                                State_Diag%Archive_AODSOAfromAqIsopWL1  .or. &
                                State_Diag%Archive_AODSOAfromAqIsopWL1  .or. &
                                State_Diag%Archive_AODDust              .or. &
                                State_Diag%Archive_AODDustWL1           .or. &
                                State_Diag%Archive_AODDustWL2           .or. &
                                State_Diag%Archive_AODDustWL3               )
```
Logically I believe two of the three `Archive_AODSOAfromAqIsopWL1` should be `Archive_AODSOAfromAqIsopWL2` and `Archive_AODSOAfromAqIsopWL3`. It does not impact simulations, and can be easily fixed supposing my guess is right.

**Comment by yantosca** [comment]
Thanks @FeiYao-Edinburgh.  This is indeed a bug.  It won't affect any simulations unless you are running RRTMG (since WL2 and WL3 are not used unless RRTMG is turned on).  We can fix this for 13.0.0.

**Comment by lizziel** [comment]
A fix for this bug went in as part of the RRTMG diagnostic updates for 13.0.0, so this is all set.
